src/main/java/service/impl/AddOperation.java

```diff
@@ -0,0 +1,17 @@
+package service.impl;
+
+import db.Storage;
+import model.Fruit;
+import model.dto.FruitRecordDto;
+import service.FruitOperationsService;
+
+public class AddOperation implements FruitOperationsService {
+    @Override
+    public int apply(FruitRecordDto fruitRecordDto) {
+        Fruit currentFruit = new Fruit(fruitRecordDto.getFruitName());
+        int currentQuantityInStock = Storage.fruits.get(currentFruit);
```

Comment:
what you will get if you call get(not existing in db fruit)? You can use getOrDefault(0);